tests/test_client.py

```diff
@@ -378,13 +378,6 @@ def test_socks_proxy(testcase, http2):
     )
 
 
-def test_simple_transport(sentry_init):
```

Comment:
Do we still support creating a custom transport by subclassing the `Transport` class? If yes, we should add a test for that functionality unless we have one already.

Reply:
we have a test_transport_option so this is covered.